### PR TITLE
Non functional updates

### PR DESCRIPTION
**Pull request by MattWellie**
# Fixes

  - README is long, images are meh, documentation in general is pretty outdated

## Proposed Changes

  - updates all URLs to point to the new talos-named repository
  - cuts out most of the README, removes fluff
  - Better documents how to add new content

The intention here is to have a better groomed README for https://populationgenomics.github.io/talos/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,165 +69,39 @@ Talos consists of the following components:
...
-only when present in the family with evidence of being _de novo_. This leverages the built-in [_de
-novo_ method in Hail](
-https://hail.is/docs/0.2/methods/genetics.html#hail.methods.de_novo), which is itself an implementation
-of [Kaitlin Samocha's _de novo_ caller](https://github.com/ksamocha/de_novo_scripts). This implementation doesn't
-naively accept the trio genotypes, but also applies some probability modelling to the genotype likelihoods, and searches
-for alt-supporting reads at low levels in parents, before treating a _de novo_ variant call as validated.
+See the companion [ClinvArbitration](https://github.com/populationgenomics/ClinvArbitration) repository for more details.
 
-### Category 5
+In Talos whenever we check for 'Pathogenic/Likely Pathogenic' in ClinVar, we use our in-house re-summary of ClinVar entries, not the standard ClinVar results. This re-aggregation of ClinVar submissions uses altered heuristics favouring decisions, instead of the default logic which cautiously assigns "conflicting interpretations of pathogenicity" unless there is perfect harmony among all submissions.
```
Comment (cassimons) [inline README.md:97]:
```suggestion
Talos uses ClinVar submissions to determine if a variant has been previously reported as Pathogenic or Likely Pathogenic. For the purpose of a Talos analysis, when multiple submissions for the same variant have conflicting classifications, we would prefer to favour the classifications provided by high-quality recent submitters. To enable this, we have ClinvArbitration, a re-aggregation of ClinVar submissions uses altered heuristics favouring decisions. This avoids the the default logic used by clinvar which cautiously assigns "conflicting interpretations of pathogenicity" unless there is perfect harmony among all submissions.
```

File: README.md [hunk 2]
```diff
@@ -58,165 +69,39 @@ Talos consists of the following components:
...
 
-![CategoryBoolean5](design_docs/images/Category5.png)
+As well as using the top-line rating, we also annotate the pathogenic SNVs with VEP, and then re-index the results on `Protein & Codon`. This enables us to assign a `PM5` evidence category to missense variants - "a missense variant at this same codon is pathogenic in clinvar'. This is the basis of the `PM5` category.
```
Comment (cassimons) [inline README.md:99]:
```suggestion
In addition to providing the top-line rating, ClinvArbitration also re-indexes ClinVar variants based on the transcript and codon they alter. Talos uses this table to identify any missense variants impacting a codon previously reported to be altered by a pathogenic variant in ClinVar and assign the `PM5` evidence category.
```

File: README.md [hunk 3]
```diff
@@ -58,165 +69,39 @@ Talos consists of the following components:
...
-re-indexed results at runtime to apply the PM5 category.
-
-This category is applied in the form `categorydetailsPM5=27037::1+27048::1`
-
-- this is a `+` delimited list of entries (can be null)
-- each entry is `ClinVar allele ID` :: `ClinVar Star rating`
-
-This is processed upon variant ingestion, and back filtered to remove any associated ClinVar IDs which are this exact
-variant.
+Resources to do this are periodically updated in the ClinvArbitration repository's Release page.
```
Comment (cassimons) [inline README.md:101]:
```suggestion
ClinvArbitration is used to re-process ClinVar releases periodically and the ready-to-use results are available via the ClinvArbitration repository's Release page.
```

**Review by cassimons — APPROVED** [review]
Looks good. A few suggestions for the main page. I will work my way through all of the docs sometime soon.

File: README.md [hunk 4]
```diff
@@ -2,50 +2,61 @@
...
-    * Optionally a VEP-annotated VCF, this path is still under evaluation
+    * Talos uses Hail Query, a PySpark-based query engine, to perform highly parallelised analysis. this is done using a proprietary Hail MatrixTable format
+    * If you have VEP-annotated VCF, a script is provided to convert this to a MatrixTable.
 
 2. ClinVar data as generated by ClinvArbitration, both the `clinvar_decisions` and `clinvar_pm5` Hail Tables. This is
    available from
-   the [ClinvArbitration Release Page](https://github.com/populationgenomics/ClinvArbitration/releases).
+   the [ClinvArbitration Release Page](https://github.com/populationgenomics/ClinvArbitration/releases), or can be generated using the code and process described in the [ClinvArbitration repository](https://github.com/populationgenomics/ClinvArbitration).
 
 3. A PED file, describing the pedigree of the participants in the study.
```
[the remark this answers is not in the document]
Comment (MattWellie) [reply README.md:43]: yeah, I was thinking about splitting this whole section into PED + PhenoPackets. I don't know how many people are actually using phenopackets, but it's a defensible choice

**Pull request merged**